llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -279,8 +280,10 @@ class Triple {
     Amplification,
     OpenCL,
     OpenHOS,
+    Kernel,
+    Mlibc,
```

Comment:
I don't know how Mlibc is intended to be used but LLVM LTO warns about differing target triples.